### ISSUE TITLE
Add user language to database

### ISSUE THREAD

**Issue by P1otrulla** [EternalCodeTeam/EternalCore#579]
The user object has not been saved in the database for a long time because there is no implementation for it. This is not a bug, the only problem is that the language set by the player is always set to default after restarting the server.

**Comment by itsevelynn21** [comment]
Hello vLuckyyy, could I help you fix this issue or any other issues in this project?

**Comment by vLuckyyy** [comment]
> Hello vLuckyyy, could I help you fix this issue or any other issues in this project?

Yes! You can create PullRequest, if you need any questions you can join to team discord.
